.github/workflows/windows_vs2022_release.yml

```diff
@@ -506,6 +512,7 @@ jobs:
     name: Pytorch Layer Tests
     needs: [ Build, Smart_CI ]
     uses: ./.github/workflows/job_pytorch_layer_tests.yml
+    if: ${{ fromJSON(needs.smart_ci.outputs.affected_components).PyTorch_FE.test }}
```

Comment:
@mvafin, could you take a look here and below, looks like a small Smart CI gap I'd like to fix there as well